api/v1alpha1/volumesnapshotbackup_types.go

```diff
@@ -27,22 +27,24 @@ type VolumeSnapshotBackupSpec struct {
 	// Namespace where the Velero deployment is present
 	ProtectedNamespace string `json:"protectedNamespace,omitempty"`
 	// Restic Secret reference for given BSL
-	ResticSecretRef corev1.LocalObjectReference `json:"resticSecretRef,omitempty"`
+	ResticSecretRef corev1.LocalObjectReference `json:"resticSecretRef,omitempty"` // override description to avoid showing TODO? https://pkg.go.dev/k8s.io/api/core/v1#LocalObjectReference
```

Comment:
Saw Tiger updating kubebuilder in Velero, wanted to know if it is possible to change description of key of object with it